src/components/NavBar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './NavBar.css';
+import userLogo from '../assets/user.png';
+
+const NavBar = () => (
+  <div className="wrapper">
+    <nav className="nav-bar">
+      <p>Bookstore CMS</p>
+      <ul className="links-nav">
+        <li><Link to="/books"> Books </Link></li>
```

Comment:
- [ ] You need to change this to attribute to match the route change in the app component as well.